app/controllers/concerns/taxonomy_navigation.rb

```diff
@@ -1,23 +1,25 @@
 module TaxonomyNavigation
-  def should_present_taxonomy_navigation?
-    tagged_to_world_wide_taxonomy? ||
-      (!content_is_tagged_to_browse_pages? &&
-      content_is_tagged_to_a_taxon?)
+  def should_present_taxonomy_navigation?(content_item)
+    if content_item.present?
```

Comment:
I find it a bit weird to nest a boolean logic chain inside an `if` statement. As I commented before, I personally prefer the line-by-line style of writing this, otherwise if you insist on using operators like `||` and `&&`, shouldn't you be consistent and lump this on as an `&&` condition?